dbms/src/Storages/Transaction/ApplySnapshot.cpp

```diff
@@ -276,6 +276,7 @@ std::vector<DM::ExternalDTFileInfo> KVStore::preHandleSnapshotToFiles(
     }
     catch (DB::Exception & e)
     {
+        LOG_INFO(log, "!!!! [error={}]", e.displayText());
```

Comment:
do we need this log? I think the log is print by `tryLogCurrentException` in ProxyFFI.cpp

Reply:
thats quite embarrassing